File: app/src/main/java/com/example/paint_diary/Data/Room.java
package com.example.paint_diary.Data;

import java.util.ArrayList;
import java.util.List;

public class Room {
    private int room_idx; // 룸 ID
    private List userList; //유저 리스트
    private String roomName; // 방 이름


    public int getRoom_idx() {
        return room_idx;
    }

    public void setRoom_idx(int room_idx) {
        this.room_idx = room_idx;
    }

    public List getUserList() {
        return userList;
    }

    public void setUserList(List userList) {
        this.userList = userList;
    }

    public String getRoomName() {
        return roomName;
    }

    public void setRoomName(String roomName) {
        this.roomName = roomName;
    }





}

File: app/src/main/java/com/example/paint_diary/Data/Room.kt
package com.example.paint_diary.Data

 class Room {

     var room_datetime: String
     var message: String
     var room_name: String
     var room_profilePhoto: String

     constructor(_room_datetime: String, _message: String, _room_name: String, _room_profilePhoto: String) {
         room_datetime = _room_datetime
         message = _message
         room_name = _room_name
         room_profilePhoto = _room_profilePhoto
     }
 }

